LocalFeeder/FeederSimulator.py

```diff
@@ -801,3 +816,21 @@ def apply_inverter_control(self, inv_control: InverterControl):
 
         self.set_properties_to_inverter(inverter, inv_control)
         return inverter
+
+    def get_incidences(self) -> IncidenceList:
+        """Get Incidence from line names to buses."""
```

Comment:
Just checking - what is the use-case of this? Will incidences from Lines (and not Transformers etc.) be sufficient?

Reply:
It was requested for someone using using their own network approximation when a Y-matrix was insufficient. They did want transformers, so I included that.